SPEC.md

```diff
@@ -9209,6 +9210,165 @@ Example output:
 </p>
 </details>
 
+### ✨ `contains_key`
+
+```
+Map[P, Y], P), Boolean contains_key(Map[P?, Y], P?)
```

Comment:
Looks like a bad merge here. Check the original diff: https://github.com/openwdl/wdl/compare/cf597dc07eb88d2fe727bbeb10473569f7b412d8..1af834fa053360f9c84dea734abcfc6bf202bb0c